lib/resolver.js

```diff
@@ -40,8 +41,12 @@ export default class PodletClientResolver {
             logger: options.logger,
             http,
         });
-        this.#fallback = new Fallback({ ...options, http });
-        this.#content = new Content({ ...options, http });
+        const httpClient = new HttpClient({
+            throwOn400: false,
+            throwOn500: false
+        })
```

Comment:
With this I suppose we never get to the code branch where we create the `HttpClient` inside `resolver.content` and `resolver.fallback` (it's always(?) truthy).

Reply:
I didn't change any behaviour here as I can see, we did instantiate the `http` in the previous version as well.
I think the intention with the HTTP client being passed in as options in `.fallback` and `.content` was to enable unit testing them easier.